src/components/FindWallet/WalletFilterSidebar.tsx

```diff
@@ -0,0 +1,122 @@
+import { useTranslation } from "next-i18next"
+import {
+  Box,
+  Flex,
+  Stack,
+  Tabs,
+  type TabsProps,
+  Text,
+  useTheme,
+} from "@chakra-ui/react"
+
+import { WalletFilter } from "@/lib/types"
+
+import WalletFilterFeature from "@/components/FindWallet/WalletFilterFeature"
+
+import { walletsListingCount } from "@/lib/utils/wallets"
+
+import { ResetFiltersButton } from "./ResetFiltersButton"
+
+export type WalletFilterSidebarProps = Omit<TabsProps, "children"> & {
+  filters: WalletFilter
+  resetWalletFilter: React.MutableRefObject<() => void>
+  resetFilters: () => void
+  setFilters: React.Dispatch<React.SetStateAction<WalletFilter>>
+  selectedPersona: number
+  setSelectedPersona: React.Dispatch<React.SetStateAction<number>>
+  updateFilterOption: (key: any) => void
+  updateFilterOptions: (keys: any, value: any) => void
+  showMobileSidebar?: boolean
+}
+
+const WalletFilterSidebar = ({
+  filters,
+  resetWalletFilter,
+  resetFilters,
+  setFilters,
+  selectedPersona,
+  setSelectedPersona,
+  updateFilterOption,
+  updateFilterOptions,
+  top,
+  showMobileSidebar,
+  ...tabsProps
+}: WalletFilterSidebarProps) => {
+  const theme = useTheme()
+  const { t } = useTranslation("page-wallets-find-wallet")
+
+  return (
+    <Tabs
+      transition="0.5s all"
+      sx={{
+        scrollbarWidth: "thin",
+        scrollbarColor: `${theme.colors.lightBorder} ${theme.colors.background}`,
+
+        "::-webkit-scrollbar": {
+          width: 2,
+        },
+        "::-webkit-scrollbar-track": {
+          bg: "background.base",
+        },
+        "::-webkit-scrollbar-thumb": {
+          bgColor: "lightBorder",
+          borderRadius: "base",
+          border: "2px solid",
+          borderColor: "background.base",
+        },
+      }}
+      {...tabsProps}
+    >
+      <Box position="sticky" top={top ?? 0} zIndex={2} w={{ md: "330px" }}>
```

Comment:
Similar to above, @nloureiro could we shift towards the presets we have available? A width of `"xs"` would be 20rem (320px)?

<img width="242" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/2b2b83e6-e620-43c2-a09b-dedfaa118fd5">